src/Pages/Saksoversikt/Saksfilter/Virksomhetsmeny/Virksomhetsmeny.tsx

```diff
@@ -124,10 +132,14 @@ export const Virksomhetsmeny = ({
                 onChange={onCheckboxGroupChange}
             >
                 <ul className="sak_virksomhetsmeny_hovedenhetliste">
-                    {organisasjonstre.map(({ hovedenhet, underenheter }) => {
+                    {organisasjonstre.map(({ hovedenhet, underenheter }, indeks) => {
                         if (søketreff && !søketreff.has(hovedenhet.OrganizationNumber)) {
                             return null;
                         }
+                        if (søkeordState === '' && indeks > visAntall && !visAlle) {
```

Comment:
hvordan funker `indeks > visAntall` mtp hoved og underenheter. indeks ser ut til å være indeks på hovedenhet, og tar kanskje ikke høyde for antall underenheter?